corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -137,6 +137,22 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         return curr;
     }
 
+    function isStyleNPerRowRepeat(styleStr) {
+        const pattern = new RegExp(`\\d+${constants.PER_ROW_REPEAT}(\\s|$)`);
+        const matchingStyles = getMatchingStyles(pattern, styleStr);
+        return matchingStyles.length > 0;
+    }
+
+    function getNPerRowStyleFromRepeatStyle(style) {
+        var parts = style.split('-');
+        var integerPart = parts[0];
+        return integerPart + '-per-row';
+    }
+
+    function isNPerRowRepeatElement(element) {
+        return element.type === constants.GROUP_TYPE && element.hasOwnProperty('style') && element.style.hasOwnProperty('raw') && isStyleNPerRowRepeat(element.style.raw)
```

Comment:
FYI looks like this is erroring on staging:
![image](https://github.com/dimagi/commcare-hq/assets/2367539/9509693d-8931-4030-a1b7-b2f658d7cbb5)
I suspect its that `element.style` exists, but is `null`.

Reply:
Thanks @esoergel for highlighting this. In local testing I had not come across this scenario where style exists but is null. Anyhow, I have pushed the fix for it and issue seems to be resolved.